### ISSUE TITLE
Robinson projection (ESRI:54030) test failed

### ISSUE THREAD

**Issue by Yaqiang** [locationtech/proj4j#87]
Following test for Robinson projection failed.
```
    @Test
    public void testRobinson() {
        checkTransform("+proj=latlong +datum=WGS84", -30, 40, "+proj=robin +datum=WGS84", -2612095.95, 4276351.58, 2e-1);
        checkTransformFromWGS84("ESRI:54030", -30., 40., -2612095.954698802, 4276351.583838239);
        checkTransformToWGS84("ESRI:54030", -2612095.954698802, 4276351.583838239, -30., 40., 1E-4);
    }
```

**Comment by Yaqiang** [comment]
I have solved the above bug and commit it to my proj4j forking repository (https://github.com/Yaqiang/proj4j/commit/42fa692c9d4b600be9bd91fff2819dfe91b7ba65) refering to the code of PROJ (https://github.com/OSGeo/PROJ/blob/master/src/projections/robin.cpp). But my last PR #86 was not reviewed and accepted, so I don't know to to commit a new PR now.

**Comment by Yaqiang** [comment]
Solved by PR #86
